Cartfile

```diff
@@ -1 +1 @@
-github "ReactiveCocoa/ReactiveCocoa" "v3.0-beta.8"
+github "ReactiveCocoa/ReactiveCocoa" "swift2"
```

Comment:
Can you target the `v4.0-alpha.1` tag or a specific commit instead. Pinning to branch heads is super fragile for consumers - see neilpa/Rex#31 where I learned my lesson.